internal/discovery/foreign-cluster-operator/foreign-cluster-controller.go

```diff
@@ -178,6 +179,10 @@ func (r *ForeignClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			klog.Error(err)
 			return ctrl.Result{}, err
 		}
+	default:
```

Comment:
Can we constrain the possible inputs also at the resource level?

Reply:
Sure, this value is returned by [this](https://github.com/liqotech/liqo/blob/24f23f23c589d68731f3e3e48d70fdcb9e5ec254/internal/discovery/foreign-cluster-operator/peeringPolicy.go#L15) function. It can only be one of the two values define, this default case is mostly to avoid forgetting to add a case if we will add other values